src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -5,7 +5,7 @@ <h1 id="top" class="color-orange">Getting Started</h1>
   </p>
   <p>
     To get started using TEAMMATES, follow the following steps, or watch our <a href="https://www.youtube.com/embed/mDtfmNmRwBM?autoplay=1&rel=0" target="_blank"><i class="fas fa-film"></i> Video Tour</a>.<br>
-    For more help, browse the answers to some <a routerLink="../instructor">frequently asked questions</a>.
+    For more help, browse the answers to some <a routerLink="/{{instructorHelpPath}}">frequently asked questions</a>.
```

Comment:
Why not just:
```html
<a [routerLink]="instructorHelpPath">
```

Reply:
Likely because your `instructorHelpPath` doesn't have a leading `/`?

Reply:
I tried this, the link changes from `http://localhost:4200/web/front/help/getting-started` to `http://localhost:4200/web/front/help/getting-started/web/front/help/instructor` and shows page not found error

Reply:
Made the changes!